src/ensembl/src/content/app/browser/Browser.tsx

```diff
@@ -107,10 +107,13 @@ export const Browser: FunctionComponent<BrowserProps> = (
   useEffect(() => {
     const { path, params } = props.match;
     const newChrLocationStr = getChrLocationStr(props.chrLocation);
-    const newUrl = path
-      .replace(':species', params.species)
-      .replace(':objSymbol', params.objSymbol)
-      .replace(':location', newChrLocationStr);
+    const newUrl =
+      path
+        .substring(0, path.length - 1)
+        .replace(':species', params.species)
+        .replace(':objSymbol', params.objSymbol) +
+      '?region=' +
+      newChrLocationStr;
 
     props.history.replace(newUrl);
```

Comment:
with `ConnectedReactRouter` this should be done via the `replace` method imported from `connected-react-router`

Reply:
That's done.